modules/disable-admission-controller-enforcement.adoc

```diff
@@ -1,16 +1,30 @@
 // Module included in the following assemblies:
 //
-// * operating/manage_security_policies/about-security-policies.adoc
+// * operating/manage_security_policies/use-admission-controller-enforcement.adoc
 :_mod-docs-content-type: PROCEDURE
 [id="disable-admission-controller-enforcement_{context}"]
-= Disabling admission controller enforcement
+= Disabling admission controller enforcement on a cluster
 
 [role="_abstract"]
-You can disable admission controller enforcement from the *Clusters* view on the {product-title} ({product-title-short}) portal.
+You can disable admission controller enforcement on a cluster when installing {product-title-short}. For clusters that you did not install by using the Operator or Helm, you can disable admission controller enforcement from the *Clusters* view on the {product-title} ({product-title-short}) portal. 
 
 .Procedure
-. In the {product-title-short} portal, select *Platform Configuration* -> *Clusters*.
-. Select an existing cluster from the list.
-. Turn off the *Enforce on Object Creates* and *Enforce on Object Updates* toggles in the *Dynamic Configuration* section.
-. Select *Next*.
-. Select *Finish*.
+. For a cluster that was installed by using the Operator, in the `SecuredCluster` custom resource (CR), edit the `admissionControl.enforcement` parameter to `Disabled`.
+. For a cluster that was installed by using Helm, in the `values-public.yaml` file, set the `admissionControl.enforce` value to `false` and run the following command:
++
+[source,terminal]
+----
+helm upgrade -n stackrox \
+  stackrox-secured-cluster-services rhacs/secured-cluster-services \
+  --reuse-values \
+  -f /config/yaml/values-public.yaml \
+  -f /config/yaml/values-private.yaml
+----
+. For clusters that are not managed by the Operator or Helm, you can use the {product-title-short} portal to change this setting:
+.. In the {product-title-short} portal, select *Platform Configuration* -> *Clusters*.
```

Comment:
In https://issues.redhat.com/browse/ROX-28753 Sakshi says this is not the recommended way to turn off enforcement. Is that correct? She wanted it noted in the docs. @clickboo  She says:

`We can disable admission controller enforcement from the Clusters view in the RHACS portal, which is NOT a recommended method.`